samples/nrf9160/aws_fota/src/main.c

```diff
@@ -152,7 +152,7 @@ void mqtt_evt_handler(struct mqtt_client * const c,
 	int err;
 
 	err = aws_fota_mqtt_evt_handler(c, evt);
-	if (err > 0) {
+	if (err == 0) {
```

Comment:
I would put these changes in the previous commit. This so to avoid non-working commits.

Reply:
I'll squash them once #1663 is merged.

Reply:
Squashed.